velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -79,6 +79,26 @@ class E2EFilterTest : public E2EFilterTestBase {
     writer_->close();
   }
 
+  void writeToMemoryAbort(const std::vector<RowVectorPtr>& batches) {
+    auto sink = std::make_unique<MemorySink>(
```

Comment:
`WriteFileSink` or `LocalFileSink` is used in the HiveDataSink. I don't think `MemorySink` is ever used. Can we add a test with LocalFileSink?